frontend/app/templates/admin/permissions/system-roles.hbs

```diff
@@ -1,155 +1,216 @@
-            <td>None</td>
-            <td>None</td>
-            <td>None</td>
-            <td>None</td>
-            <td>None</td>
-            <td>None</td>
-            <td>None</td>
-            <td>None</td>
-            <td>None</td>
-            <td>None</td>
-            <td>None</td>
-            <td>None</td>
-            <td>None</td>
-            <td>None</td>
-            <td>None</td>
-            <td>None</td>
-        </tr>
-    </tbody>
+<table class="ui celled structured table">
+	<thead>
+		<tr>
+			<th>Permissions</th>
+			<th>Super Admin</th>
+			<th>Admin</th>
+			<th>Sales</th>
+			<th>Verified User</th>
+			<th>Unverified User</th>
+			<th>Anonymous User</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td>Access to the homepage</td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+		</tr>
+		<tr>
+			<td>Access to the homepage but can change details also like Email</td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td></td>
+			<td></td>
+		</tr>
+		<tr>
+			<td>Create and edit own badges and other user badges</td>
+            <td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td></td>
+			<td></td>
+			<td></td>
+			<td></td>
+		</tr>
+		<tr>
+			<td>Create and edit own badges </td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+		</tr>
+		<tr>
+			<td>Can view and edit settings</td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+		</tr>
+		<tr>
+			<td>Can view and edit settings of other users</td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td></td>
+			<td></td>
+			<td></td>
+			<td></td>
+		</tr>
+		<tr>
+			<td>Can view & edit password of own as well as other users on system</td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td><i class="large green checkmark icon"></i></td>
+			<td></td>
+			<td></td>
+			<td></td>
```

Comment:
Empty table data tags? Why do you do that?

Reply:
Actually, This was to line keep linings and keep UI clean .

Reply:
I don't think you'd require that.

Reply:
![screenshot from 2018-10-01 18-21-56](https://user-images.githubusercontent.com/17084322/46345534-91cc9a80-c662-11e8-9864-5e1c637b4feb.png)
See the image the empty blocks has lines due to empty tags else it would be empty.